packages/bpk-component-info-banner/src/BpkInfoBannerInner.tsx

```diff
@@ -91,23 +91,14 @@ type ToggleButtonProps = {
   expanded: boolean;
 };
 
-const ToggleButton = (props: ToggleButtonProps) => {
-  const classNames = getClassName('bpk-info-banner__expand-icon');
-
-  return (
-    <button
-      type="button"
+const ToggleButton = (props: ToggleButtonProps) => (
+    <div
       className={getClassName('bpk-info-banner__toggle-button')}
-      aria-label={props.label}
-      aria-expanded={props.expanded}
       title={props.label}
     >
-      <div className={classNames}>
-        {props.expanded ? <CollapseIcon/> : <ExpandIcon/> }
-      </div>
-    </button>
+      {props.expanded ? <CollapseIcon /> : <ExpandIcon />}
```

Comment:
We can add a <span className="visually-hidden">Icon text</span> or add altText to the icon to have 'view more' added.